fastlane/lib/fastlane/actions/docs/upload_to_play_store.md

```diff
@@ -83,6 +83,18 @@ Expansion files (obbs) found under the same directory as your APK will also be u
 - they are identified as type 'main' or 'patch' (by containing 'main' or 'patch' in their file name)
 - you have at most one of each type
 
+If you want only update expansion files from previous version on Google Play use
+
+```no-highlight
+fastlane supply --apk path/app.apk --obb_main_references_version version --obb_main_file_size size
```

Comment:
Should `version` and `size` replaced by a real value here or is this the literal command that should be used?

Reply:
It should be replaced by real values, i.e. fastlane supply --apk path/app.apk --obb_main_references_version 21 --obb_main_file_size 666154207

Reply:
Ok, I think you can just use these as sample values then as we usually just do it that way in the docs.